cmd/config/notify/legacy.go

```diff
@@ -11,17 +11,16 @@ import (
 
 // SetNotifyKafka - helper for config migration from older config.
 func SetNotifyKafka(s config.Config, kName string, cfg target.KafkaArgs) error {
+	if !cfg.Enable {
```

Comment:
Would it be a big thing to change the function signatures to `MigrateNotifyKafka` so it is clear that all of these are for migrating? Since they are exported I feel it would be an easy mistake to use these for setting the config?

With a decent editor I wouldn't expect it to be a big change.

Reply:
These are only legacy functions and never used. So I will keep them as is